### DISCUSSION TITLE
Welcome to terraform-provider-bitbucket Discussions!

### DISCUSSION THREAD

**Discussion by DrFaust92** [DrFaust92/terraform-provider-bitbucket#142] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by reuben-james** [comment]
Hello there, I'm wondering if the module supports the ability to ensure that user/group permissions on repositories are removed if they are not in the Terraform configuration. 

For example, if I apply a list of groups and their permissions, apply the configuration, and then an admin gives an individual user/group ad-hoc permissions to a repository, when the Terraform is applied again, they should be removed as not present in the Terraform config.
